exampleCourse/questions/template/multipleChoice/multipleChoiceMultipleCorrect/README.md

```diff
@@ -0,0 +1 @@
+This multiple-choice question does not include a server.py. Students are asked to select the colors that can be found in a rainbow. The question randomly selects one of the 7 colors of the rainbow as the correct answer, and 3 other distractors.
```

Comment:
Some pedantic person is going to say that navy, gold, lime, and aqua can indeed be found in the continuous spectrum on the rainbow. Let's think about if there's a better premise for a question demonstrating a multiple-choice question with multiple correct answers.